tasks/main.yml

```diff
@@ -57,7 +57,7 @@
       (apparmor__enabled|d() | bool) and
       (ansible_cmdline.apparmor|d(0)|string == "1") and
       (ansible_cmdline.security|d("none") == "apparmor") }}'
-  always_run: True
+  check_mode: no
```

Comment:
Please use `False` instead of `no` to be consistent with the rest of DebOps codebase.